climpred/metrics.py

```diff
@@ -2168,6 +2176,287 @@ def _crpss_es(forecast, verif, dim=None, **metric_kwargs):
 )
 
 
+def _discrimination(forecast, verif, dim=None, **metric_kwargs):
```

Comment:
In `xskillscore` for this and `reliability`, the description is that it returns the data to construct the "discrimination/reliability diagram." Might be worth mentioning as a note here. For now, the user can construct it themselves, but could have a convenience function for that later.

Reply:
i though about showing how to plot the data in the examples but then I would need to load all the data also. too much work.